projects/canopy/src/styles/mixins.scss

```diff
@@ -13,6 +13,15 @@
   box-shadow: 0 0 0 0.25rem $bg-color;
 }
 
+// this mixin will replace lg-outer-focus-outline in the future when all the components will have to use this focus styling
+@mixin lg-new-outer-focus-outline(
```

Comment:
As per previous comment could we name this `lg-focus-outline`?

Reply:
Could you also update the notes for the mixins to list this new one and give it a description?

Reply:
> @st3fun1 great work on this 👏
> 
> One thing I've noticed is that if you do a continuous click on the buttons the behaviour on the `button` tag is slightly different from the `anchor` tag:
> 
>  btns.mov 
> I think it happens for both primary and secondary buttons.

I saw in the code that the links are treated a bit differently for this scenario.

Reply:
I don't think so, I think they should behave the same 🤔  not sure why we did it slightly different before 🤷‍♀️